layers/+lang/d/packages.el

```diff
@@ -72,4 +72,4 @@
   (spacemacs/counsel-gtags-define-keys-for-mode 'd-mode))
 
 (defun d/post-init-helm-gtags ()
```

Comment:
why not delete the enclosing function as well (the same applies to other changes)

Reply:
Removing the enclosing function will got the following warning:
```
(Spacemacs) Warning: package helm-gtags not initialized in layer go, you may consider removing this package from the package list or use the :toggle keyword instead of a `when' form.
```

This warning is generated here: https://github.com/syl20bnr/spacemacs/blob/develop/core/core-configuration-layer.el#L914

The function `go/post-init-helm-gtags` needs to be bound in language layers. `configuration-layer/make-package` will check that.

Or do I need to add `:toggle` keyword for `helm-gtags` in the package list?

Reply:
simply remove helm-grabs from the <layer>-package variable defined in the beginning of the packages.el files